savaslabs/savaslabs.github.io
# Feature/add durham

Fixes issue (Ben doesn't use issues yet)

## Summary of changes

1. Adds a /durham page featuring the #moffice

## Notes

This was primarily borrowed from our `/services` page and repurposed 

@AnneTee you should change things as you see fit. Considerations:

- I repurposed the client quote component for the TriDUG callout - but clearly it has nothing to do with a quote so changing the markup or used styles there is welcome.
- I created a `_durham.scss` file, but may not be the way it should be structured

## To test

- @AnneTee : See if you would make changes to structure
- @beneckerson : Verify it on the [staging site](https://savas-staging.netlify.com/durham/)
